cves/kernel/CVE-2021-45486.yml

```diff
@@ -141,10 +148,10 @@ unit_tested:
 
     For the fix_answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  code:
```

Comment:
The VCC with commit 6da2ec56059c3c7a7e5f729e6349e74ace1e5c57 seems like it is just a refactoring. Can you check this out?